### ISSUE TITLE
Meshtastic on T-Watch S3

### ISSUE THREAD

**Issue by macflyhub** [Xinyuan-LilyGO/TTGO_TWatch_Library#252]
The amount of interest in having the watch run Meshtastic and to be able to send a message is huge. (Pity no GPS)
It appears that nobody can get it working properly without big battery drain, crashing and functional issues.
I know it isn't related specifically to this code but there is no better place to mention it and hopefully someone will take note!

**Comment by RangerMauve** [comment]
Yeah it'd be nice if the core meshtastic functions were available via a library with finer control over when the network code is awake for the application. I imagine something like only configuring the meshtastic code to run "on demand" when the watch is awake would be helpful.
